packages/cli/src/lib/implementation/only-plugins-utils.ts

```diff
@@ -26,13 +29,14 @@ export function filterCategoryByOnlyPluginsOption(
       const isNotSkipped = onlyPlugins.includes(ref.slug);
 
       if (!isNotSkipped) {
-        console.info(
-          `${chalk.yellow('⚠')} Category "${
-            category.title
-          }" is ignored because it references audits from skipped plugin "${
-            ref.slug
-          }"`,
-        );
+        verbose &&
+          console.info(
+            `${chalk.yellow('⚠')} Category "${
+              category.title
+            }" is ignored because it references audits from skipped plugin "${
+              ref.slug
+            }"`,
+          );
       }
```

Comment:
Please use `if`-statements for conditional execution, logical operators like `&&`, `||`, `!` or `? :` are intended for expressions (i.e. something that has returns a value).

In fact, you can merge the `verbose` flag into the `if`-statement above:

```ts
if (!isNotSkipped && verbose) {
  console.info(...);
}
```

Reply:
yes. way better!

Also thx for the additional info on logical operators and where to use them